updater

```diff
@@ -557,27 +558,24 @@ update_now_gui() { #input: updatable_files and updatable_apps variables
       done
     fi
   }
-  #update any files with terminal_manage. if it fails terminal-run will update the files in the background. This is a safety measure to update pi-apps scripts if terminal fails to launch.
+  
+  #contruct a queue to send to terminal_manage_multi.
   if [ ! -z "$updatable_files" ]; then
-    queue+="update $(echo "filelist:$updatable_files" | tr '\n' ':' | sed "s,:$,,")"$'\n'
+    queue+="update filelist:$(echo -n "$updatable_files" | tr '\n' ':')"$'\n'
   fi
-
-  if [ ! -z "$refresh_apps" ];then
-    for app in $refresh_apps ;do
-      # if app is a compressed update app, do a refresh and not an update
-      queue+="refresh $app"$'\n'
-    done
+  
+  if [ ! -z "$refreshable_apps" ];then
+    queue+="$(echo "$refreshable_apps" | sed 's/^/refresh /g')"$'\n'
   fi
   
   if [ ! -z "$updatable_apps" ];then
-    for app in $updatable_apps ;do
-      # if app is not a compressed update app, do a regular update
-      if ! echo "${refresh_apps}" | grep -q "^$app"; then
-        queue+="update $app"$'\n'
-      fi
-    done
+    queue+="$(echo "$updatable_apps" | list_subtract "$refreshable_apps" | sed 's/^/refresh /g')"$'\n'
   fi
   queue="${queue::-1}" #remove final newline character
+  queue="$(tac <<<"$queue")" # reverse order because manage daemon reverses it again; affects alphabetization, not priority
```

Comment:
Good catch. I did not consider the tac in the daemon (which was intented only reverse the order which the pending pipes were read) also reversed the file contents themselves.